include/ceed/ceed.h

```diff
@@ -494,22 +494,32 @@ CEED_EXTERN const char *const CeedQuadModes[];
 /// @ingroup CeedBasis
 typedef enum {
   /// Line
-  CEED_LINE = 1 << 16 | 0,
+  CEED_TOPOLOGY_LINE = 1 << 16 | 0,
   /// Triangle - 2D shape
-  CEED_TRIANGLE = 2 << 16 | 1,
+  CEED_TOPOLOGY_TRIANGLE = 2 << 16 | 1,
   /// Quadralateral - 2D shape
-  CEED_QUAD = 2 << 16 | 2,
+  CEED_TOPOLOGY_QUAD = 2 << 16 | 2,
   /// Tetrahedron - 3D shape
-  CEED_TET = 3 << 16 | 3,
+  CEED_TOPOLOGY_TET = 3 << 16 | 3,
   /// Pyramid - 3D shape
-  CEED_PYRAMID = 3 << 16 | 4,
+  CEED_TOPOLOGY_PYRAMID = 3 << 16 | 4,
   /// Prism - 3D shape
-  CEED_PRISM = 3 << 16 | 5,
+  CEED_TOPOLOGY_PRISM = 3 << 16 | 5,
   /// Hexehedron - 3D shape
-  CEED_HEX = 3 << 16 | 6,
+  CEED_TOPOLOGY_HEX = 3 << 16 | 6,
 } CeedElemTopology;
 CEED_EXTERN const char *const CeedElemTopologies[];
 
+/// Type of FE space;
+/// @ingroup CeedBasis
+typedef enum {
+  /// H1 FE space
+  CEED_FE_SPACE_H1 = 1,
+  /// H(div) FE space
+  CEED_FE_SPACE_HDIV = 2,
+} CeedFESpace;
+CEED_EXTERN const char *const CeedFESpaces[];
```

Comment:
I just noticed that this isn't used in the public interface (now, and I'm not sure whether it should ever be).

Reply:
I think internally its only used in `CeedBasisView` now.

Reply:
Yeah, it is used only in `CeedBasisView` so far.